packages/supermassive/src/bin/supermassive.ts

```diff
@@ -16,7 +17,64 @@ export function supermassive(): Command {
     .action(async (files: Array<string>) => {
       await typeDefsToImplicitResolversImpl(files);
     });
-  return program.name("supermassive").addCommand(command);
+
+  const generateInterfacesCommand = new Command();
+  generateInterfacesCommand
```

Comment:
Let's do it as follows:

--input - files or glob
--output-dir - directory relative to file, default __generated__
--context-import - from where to import context
--context-name - name of context

Reply:
Maybe we should have config file in future, but no need rn.